addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -99,11 +99,15 @@ class MessageList extends Component {
             isLastMessageVisible:
                 lastMessageRef &&
                 lastMessageRef.isBottomVisible({ offset: 10 }),
-            scrollHeight: this.el.scrollHeight,
-            scrollTop: this.el.scrollTop,
+            scrollHeight: this.scrollableElement.scrollHeight,
+            scrollTop: this.scrollableElement.scrollTop,
         };
     }
 
+    mounted() {
+        this.scrollableElement = this.props.getScrollableElement() || this.el;
+    }
```

Comment:
Not in mounted, this reference should be dynamic. Create a private method here that returns the value.

Reply:
Method should do in pseudo code:
if getScrollableElement
return getScrollableElement()
else
return this.el

And remove default empty function from getScrollableElement prop